msal/tests/dev apps/XForms/XForms/CachePage.xaml.cs

```diff
@@ -111,7 +111,7 @@ public void OnExpire(object sender, EventArgs e)
             var tokenCache = App.MsalPublicClient.UserTokenCache;
 
             // set access token as expired
-            accessTokenCacheItem.ExpiresOnUnixTimestamp = GetCurrentTimestamp();
+            accessTokenCacheItem.ExpiresOnUnixTimestamp = GetCurrentTimestamp().ToString();
```

Comment:
>GetCurrentTimestamp [](start = 58, length = 19)

nit: consider making the method return a string

Reply:
changed